apps/molgenis-components/src/components/forms/BaseInput.vue

```diff
@@ -2,24 +2,44 @@
 export default {
   name: 'BaseInput',
   props: {
+    /**
+     * Unique identifier https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id
+     * Has no direct emx2 implementenation but can be constructend by concatinating the table name with the field name
+     */
```

Comment:
should we then do this automatically if 'null'? Or how do you recommend us to use this?

Reply:
it can be done 'automatically' in a higher level comp that extends or is composed using the baseComp. The usage context can 'generate' the id avoid the need for the component to generate a random id

Reply:
What intermediate do you think of? Because I would suppose we don't want to repeat this for every *Input? I.e. StringInput, IntInput etc?

Reply:
what 'intermediate' or 'repeat' do you mean, could you be more specific or supply an example ?

Reply:
sorry, I meant to ask: what do you mean by 'higher level comp' ? is that something between 'BaseInput' and 'StringInput'?

Reply:
EMXFormComponent or just FormComponent if you like